src/components/Button.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonName }) => <button type="button">{buttonName}</button>;
-
+const Button = ({ buttonName, color, wide }) => {
+  const twice = wide === true && 'wide';
+  return (
+    <button type="button" className={`button border ${twice} ${color}`}>
+      {buttonName}
+    </button>
+  );
+};
 Button.propTypes = {
   buttonName: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.bool,
 };
 
 Button.defaultProps = {
   buttonName: '',
+  color: '#fe9291',
```

Comment:
This `color` value is redundant. Can you call it `orange` or something similar and add a css declaration for it in your css. This way when no value is passed, the `orange` string is passed as a class name and styled in the css to make such buttons orange color by default. And when a color name is passed, such color name should have a corresponding css declaration to make the button such color.